src/logger.ts

```diff
@@ -0,0 +1,74 @@
+import * as winston from 'winston';
+import * as os from 'os';
+import * as grpc from 'grpc';
+import * as WinstonElasticsearch from 'winston-elasticsearch';
+import * as data from 'winston-elasticsearch/index-template-mapping.json';
+import { confLogger, serviceName } from './config';
+
+const Elasticsearch = require('winston-elasticsearch');
+
+// index pattern for the logger
+const indexTemplateMapping = data;
```

Comment:
Why though?

Reply:
to load the json object. Is there a better way to do that?

Reply:
change `data` to `indexTemplateMapping` on the import line and work with that object.
This line is not necessary.